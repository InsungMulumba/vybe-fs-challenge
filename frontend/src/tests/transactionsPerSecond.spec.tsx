/**
 * @jest-environment jsdom
 */

import * as renderer from "react-test-renderer";
import TransactionsPerSecond from "../charts/transactionsPerSecond";

jest.mock("react-apexcharts", () => ({
  __esModule: true,
  default: () => <div />,
}));

const testData = [3001, 3002, 3003, 3004, 3005, 3006, 3007, 3008];

describe("Transactions Per Secong Component", () => {
  it("renders Time Chart Series correctly", () => {
    const tree = renderer
      .create(<TransactionsPerSecond data={testData} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
