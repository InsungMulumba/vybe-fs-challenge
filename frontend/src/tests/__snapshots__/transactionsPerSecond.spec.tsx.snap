// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transactions Per Secong Component renders Time Chart Series correctly 1`] = `<div />`;
